lesson1.md

```diff
@@ -135,3 +142,360 @@ We'll be using these tools during our course:
 | `` ` `` Back tick `` ` `` | `_` Underscore `_` |
 | `/` Slash, or Forward slash `/` | `:` Colon `:` |
 | `\` Backslash `\` | `;` Semicolon `;` |
+
+---
+
+<!-- .slide: style="font-size:70%" -->
+
+### What is programming?
+
+Programming involves knowing and doing the following things:
+* input: Get data from the keyboard, a file, a sensor, or some other device.
+* output: Display data on the screen, or send data to a file or other device.
+* math: Perform basic mathematical operations like addition and division.
+* decisions: Check for certain conditions and execute the appropriate code.
+* repetition: Perform some action repeatedly, usually with some variation.
+
+(Source: http://greenteapress.com/thinkpython/thinkpython.pdf, Page 3)
+
+---
+
+<!-- .slide: id="lesson1:datatypes" -->
+
+# Basic Data Types
+
+---
+
+### Let's start with JavaScript!
+
+* Open your favorite browser
```

Comment:
I think it might be good to suggest using Chrome to reduce confusion because the different debuggers/consoles  look different